src/test/java/com/google/starfish/services/LabelServiceTest.java

```diff
@@ -0,0 +1,84 @@
+package com.google.starfish.services;
+
+import com.google.starfish.models.Label;
+import java.sql.SQLException;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import javax.sql.DataSource;
+import java.util.HashMap;
+import static com.ninja_squad.dbsetup.Operations.*;
+import static org.junit.Assert.assertTrue;
+
+import com.ninja_squad.dbsetup.DbSetup;
+import com.ninja_squad.dbsetup.destination.DataSourceDestination;
+import com.ninja_squad.dbsetup.operation.Operation;
+
+@RunWith(JUnit4.class)
+public class LabelServiceTest {
+
+  private Constants constants = new Constants();
+  private String LABELS = constants.LABELS;
+  private DataSource pool = Constants.pool;
+  private LabelService labelService = new LabelService();
+
+  private static final boolean runTests = Constants.TEST_DB_NAME.equals("starfish_test");
+
+  /** Clear DB, insert reference data, and insert 4 additional labels */
+  @Before
+  public void prepare() throws Exception {
+    if(!runTests) throw new Exception("Wrong Test Database Name");
+    Operation operation =
+        sequenceOf(
+            CommonOperations.DELETE_ALL,
+            CommonOperations.INSERT_REFERENCE_DATA,
+            insertInto(LABELS)
+                .columns("title", "type")
+                .values("cornell", "School")
+                .values("the university of florida", "School")
+                .values("se2205", "Course")
+                .values("bus1220", "Course")
+                .build());
+    DbSetup dbSetup = new DbSetup(new DataSourceDestination(pool), operation);
+    dbSetup.launch();
+  }
+
+  /** Test that a school label can be inserted */
+  @Test
+  public void testInsertSchoolLabel() throws SQLException {
+    String testSchool = "mit";
+    labelService.insertSchoolLabel(pool, testSchool);
+    Label schoolLabel = labelService.getLabelByTitle(pool, testSchool);
+    assertTrue(schoolLabel.getTitle().equals(testSchool) && schoolLabel.getType().equals("School"));
```

Comment:
Actually, I would maybe separate these into two separate `assertTrue()` statements because if the test fails it'll be easier to know which one failed by separating them out (here and below)

Reply:
Good point, I'll do that!